imports/plugins/core/ui/client/components/button/button.jsx

```diff
@@ -51,6 +51,12 @@ class Button extends Component {
     }
   }
 
+  handleKeyUp = (event) => {
+    if (event.keyCode === 32 || event.keyCode === 13) {
```

Comment:
Somewhere else in the app, you've commented what these event.keyCodes are equal to, we should add that here too for easy reference.